### DISCUSSION TITLE
Netbox 3.1.4 - Server Error - Azure AD

### DISCUSSION THREAD

**Discussion by Nino-Rey** [netbox-community/netbox#9215] [General]
Hi,

We have followed this guide for installation: https://netbox.readthedocs.io/en/develop/administration/authentication/microsoft-azure-ad/ however, I'm getting the error below.

Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'social_core.exceptions.AuthForbidden'>

Your credentials aren't allowed

Python version: 3.8.10
NetBox version: 3.1.4
If further assistance is required, please post to the NetBox discussion forum on GitHub.

After I click on azuread-oauth2 url, provide my credentials then it gets me this error page. 

![netbox_server_error](https://user-images.githubusercontent.com/93611480/165062377-45bbb202-771c-4523-a23c-643cace4c2e3.png)

I really appreciate any assistance.

**Comment by ChrisrHeimdal** [comment]
i get the exact same error when following the same guide.

**Reply by ChrisrHeimdal** [reply]
Got it working by removing the curly brackets in SOCIAL_AUTH_AZUREAD_OAUTH2_KEY and SOCIAL_AUTH_AZUREAD_OAUTH2_SECRET.
noob mistake 😆
